cmake/modules/FindASIO.cmake

```
@@ -17,7 +17,9 @@ module may serve as a useful start but you will need to modify it.
 #]=======================================================================]
 
 if(NOT WIN32)
-  message(FATAL_ERROR "ASIO is only supported on Windows.")
+  message(WARNING "ASIO is only supported on Windows.")
+  set(ASIO_FOUND OFF)
+  return()
```

Contrary to my previous (now retracted) message. This change to the PortAudio library CMake behavior does not really belong in a PR that is supposed to be specific to the C++ library. If this change is generally considered to be desirable it should be in a separate PR. Suggest rolling it back:

```suggestion
  message(FATAL_ERROR "ASIO is only supported on Windows.")
```